wpilibc/athena.gradle

```diff
@@ -183,7 +149,7 @@ publishing {
     publications {
         wpilibc(MavenPublication) {
             artifact wpilibcZip
-
+            artifact project(':hal').halZip
```

Comment:
Ugh. This should be a separate artifact. Something like edu.wpi.first.wpilib:hal.